.pre-commit-config.yaml

```diff
@@ -22,18 +22,18 @@ repos:
         args: [--remove]
 
 -   repo: https://github.com/pre-commit/mirrors-autopep8
-    rev: 'v1.5.6'  # Use the sha / tag you want to point at
+    rev: 'v1.5.7'  # Use the sha / tag you want to point at
     hooks:
     -   id: autopep8
         args: [--max-line-length=120]
```

Comment:
This didn't catch the line too long (121) in data.py (line 103)?

Reply:
Apparently a bug in autopep8. This `autopep8 --max-line-length=120 -a -d data.py` did not catch it either. I fixed it manually. Perhaps move to `black` for formatting at some point.